Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -1903,7 +1903,7 @@
       state: slug
       sprite: Mobs/Animals/slug.rsi
   - type: Carriable #DeltaV
-    freeHandsRequired: 1 
```

Comment:
untouch

Reply:
This just happens sometimes. You can't readd these spaces and have them recognized. At least I've never been ale to

Reply:
> This just happens sometimes. You can't readd these spaces and have them recognized. At least I've never been ale to

You could either revert the commit or revert to the old version of the file with `git log` and `git checkout [hash] -- path/to/file`, then only commit the lines that you want changed.

Reply:
Seems like a lot of work to preserve some necessary spaces

Reply:
Doesn't work. No matter what I do, those lines are always different
![image](https://github.com/user-attachments/assets/2830c737-f815-433d-8f2e-809ab93ce09c)

Reply:
Personally the way I'd do it (which I find a bit more intuitive) is
```
# Note the trailing tilde following the hash. In this case it'd be git checkout -p 7116a7b~ ...
git checkout -p (commit hash with bad whitespace)~ (file(s) with bad whitespace)
```
And you'd then go through and enter `y` for parts of the commit that have bad whitespace and `n` for parts that were fine. You'd then might see some warnings about you having added in trailing whitespace, which is what you want. (This in general is how I prefer to selectively undo only certain line changes of a commit.)

Reply:
So, I tried this. Seemed to work. I committed and pushed the file again and I don't see any change

Reply:
> So, I tried this. Seemed to work. I committed and pushed the file again and I don't see any change

Wrong branch Velcro!

Reply:
Hey...look at that